docs/user-guide/grpc.md

```diff
@@ -121,8 +133,24 @@ Next we will modify gRPC Client to use our Træfik self-signed certificate:
 ```go
 // ...
 
+import (
+
+// ...
+
+  "crypto/x509"
+  "fmt"
+  "google.golang.org/grpc/credentials"
+  "io/ioutil"
+
+)
+
+const (
```

Comment:
Could remove those constants and restore previous code?
You can create local variable if you want.

Reply:
original grpc-go use const why don't use it?

Reply:
please remove

Reply:
This code is for illustrative purposes only. We want this section to be small.